### PR TITLE
WIP: Upgrade TensorFlow 1.12.0 to 1.15.0 without contrib

### PR DESCRIPTION
**Pull request by maziyarpanahi**
- [x] Update TensorFlow in build.sbt to `1.15.0`
- [ ] Test current graphs generated by `1.12.0`
- [ ] Test `NerDLApproach` with `1.15.0`
- [x] Convert `tf.contrib.rnn. LSTMCell` to native Keras `tf.keras.layers. LSTMCell`
- [x] Convert `tf.nn.bidirectional_dynamic_rnn` to native Keras `tf.keras.layers.Bidirectional`, `tf.keras.layers.LSTM`, and use `tf.keras.Sequential` with masks
- [x] Convert `tf.layers.conv1d` to native Keras `tf.keras.layers.Conv1D`
- [x] Change `squeeze_dims` to `axis`
- [x] Change `self.dropout` to `rate=1-self.dropout`
- [ ] Replace .so files for mac (need to generate the .so file for Java - contrib is no longer supported)
- [ ] Replace .so files for linux (need to generate the .so file for Java - contrib is no longer supported_
- [ ] Is the contrib still an issue in windows
- [ ] Test pre-trained `NerDLModel` models in `1.12.0` in `1.15.0` (probably won't work, retraining all models required)
- [ ] Adopt Bert model to use TF natively instead of contrib if possible in 1.15.0

### CODE REVIEW

**Comment by maziyarpanahi** [comment]
Changes made in the model to adopt to `1.15.0` didn't effect the results compare to `1.12.0`:

Screenshot 2019-08-31 21 38 11

NOTE: The lowest+slowest metrics are for `non-contrib` models.
